docs/user-guide/running.md

```diff
@@ -68,7 +80,31 @@ Following step should be done after flotta-device-worker is installed on the edg
 
 Start yggdrasil with from the yggdrasil repository directory:
 
-`sudo go run ./cmd/yggd --log-level trace --transport http --cert-file /etc/pki/consumer/cert.pem --key-file /etc/pki/consumer/key.pem --client-id-source machine-id --http-server <your.k8s-ingress:8888>`
+`
+sudo yggd \
+  --protocol http \
+  --path-prefix api/flotta-management/v1 \
+  --client-id $(cat /etc/machine-id) \
+  --cert-file /tmp/cert.pem \
+  --key-file /tmp/key.pem \
+  --ca-root /tmp/ca.pem \
+  --socket-addr @yggd \
+  --server 127.0.0.1:8043
```

Comment:
should this be similar to the commands decribed here https://github.com/project-flotta/flotta-operator/blob/092392deb17c39476c2ee24fd534f2e4ae334afc/README.md#inside-the-cluster?

Reply:
I think we now use fqdn instead of IP.

Reply:
We are using 8043 port forward on step 3, I do not think so.

Reply:
yeah, I will